crates/verify/src/utils.rs

```diff
@@ -59,6 +59,14 @@ pub struct JsonResult {
     pub message: Option<String>,
 }
 
+pub fn version_without_build(version: &str) -> String {
+    version.split('+').next().unwrap_or("").to_string()
```

Comment:
this seems like it should just be an `Option<_>` since all we do is check if it's empty and then provide a better default.